ubuntu-advantage.1

```diff
@@ -55,6 +55,25 @@ cryptoapi.
 disable-fips
 It's currently not possible to disable FIPS after it has been enabled.
 
+.TP
+.B
+update-fips \fItoken\fR[\fB\-y|--yes\fR]
```

Comment:
There should be an exta space after the first `\fR` and before `[`

Reply:
Fixed.